documentation/src/main/resources/pages/ditto/release_notes_373.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Release notes 3.7.3
+tags: [release_notes]
+published: true
+keywords: release notes, announcements, changelog
+summary: "Version 3.7.3 of Eclipse Ditto, released on 31.03.2025"
+permalink: release_notes_373.html
+---
+
+This is a bugfix release, no new features since [3.7.2](release_notes_372.html) were added.
+
+## Changelog
+
+Compared to the latest release [3.7.2](release_notes_372.html), the following changes and bugfixes were added.
+
+### Bugfixes
+
+This is a complete list of the
+[merged pull requests](https://github.com/eclipse-ditto/ditto/pulls?q=is%3Apr+milestone%3A3.7.2).
```

Comment:
Link is wrong, should be: https://github.com/eclipse-ditto/ditto/pulls?q=is%3Apr+milestone%3A3.7.3